### PR TITLE
feat: oneclick deferred

### PR DESCRIPTION
**Pull request by victormendoza96**
- [x] add oneclick deferred
- [x] add error pages for oneclick deferred

### CODE REVIEW

File: resources/views/oneclick-mall-deferred/capture.blade.php [hunk 1]
```diff
@@ -0,0 +1,81 @@
+@php
+    $navigation = ['capture' => 'Capturar pago', 'other' => 'Consultas'];
+@endphp
+
+<x-layout active-link="Oneclick Mall Diferido" :navigation="$navigation">
+    <h1 id="capture">Oneclick Mall Diferido - Capturar pago</h1>
+    <p class="mb-32">
+        En este paso debemos capturar la transacción para realmente capturar el dinero que habia sido previamente
+        reservado al hacer la transacción.
```
Comment (Matiasnickolas) [inline resources/views/oneclick-mall-deferred/capture.blade.php:9]:
```suggestion
        En este paso debemos capturar la transacción para hacer efectiva la reserva de dinero realizada previamente en la etapa de autorización.
```

File: resources/views/oneclick-mall-deferred/finish.blade.php [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        <div class="divided-card">
+            <p>Después de una inscripción exitosa, tienen dos opciones: autorizar un pago o borrar al usuario que se
+                acaba de inscribir.</p>
+            <div class="card-right-container">
+                <a href={{ route('oneclick-mall-deferred.delete', ['userName' => $table['username'], 'tbkUser' => $table['tbk_user']]) }}
+                    class="tbk-button primary">
+                    BORRAR USUARIO
+                </a>
+                <a href={{ route('oneclick-mall-deferred.authorize', ['userName' => $table['username'], 'tbkUser' => $table['tbk_user']]) }}
+                    class="tbk-button primary">AUTHORIZAR UN PAGO</a>
```
Comment (Matiasnickolas) [inline resources/views/oneclick-mall-deferred/finish.blade.php:56]:
```suggestion
                    class="tbk-button primary">AUTORIZAR UN PAGO</a>
```

File: resources/views/oneclick-mall-deferred/refund.blade.php [hunk 3]
```diff
@@ -0,0 +1,65 @@
+@php
+    $navigation = ['refund' => 'Reembolsar'];
+@endphp
+
+<x-layout active-link="Oneclick Mall Diferido" :navigation="$navigation">
+
+    <h1 id="refund">Oneclick Mall Diferido- Reembolsar</h1>
```
Comment (Matiasnickolas) [inline resources/views/oneclick-mall-deferred/refund.blade.php:7]:
```suggestion
    <h1 id="refund">Oneclick Mall Diferido - Reembolsar</h1>
```

File: resources/views/oneclick-mall-deferred/start.blade.php [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+        dirigir al tarjetahabiente al formulario de inscripción.
+    </p>
+
+    <h2>Paso 1: Petición</h2>
+    <ul class="mb-32">
+        <li>Comienza por importar la librería Oneclick en tu proyecto.</li>
+        <li>Después podrás iniciar una inscripción.</li>
+    </ul>
+    <x-snippet>
+        use Illuminate\Http\Request;
```
Comment (Matiasnickolas) [inline resources/views/oneclick-mall-deferred/start.blade.php:18]: unused in this snippet

File: resources/views/oneclick-mall-deferred/start.blade.php [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+        use Transbank\Webpay\Options;
+        use Transbank\Webpay\Oneclick\MallInscription;
+        use Transbank\Webpay\Oneclick\MallTransaction;
```
Comment (Matiasnickolas) [inline resources/views/oneclick-mall-deferred/start.blade.php:21]: to start an inscription, isn't necessary import MallTransaction

File: resources/views/oneclick-mall-deferred/start.blade.php [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+
+    <h2>Paso 3: Creación del formulario</h2>
+    <p class="mb-32">
+        Utiliza estos datos de respuesta para generar y presentar un formulario de pago al Tarjetahabiente.
+        Este formulario será la interfaz a través de la cual el usuario realizará su transacción.
+    </p>
+
+    <x-snippet>
+        form action="https://webpay3gint.transbank.cl/webpayserver/initTransaction" method="POST">
+        input type="hidden" name="token_ws" value="{{ $resp->token }}" />
```
Comment (Matiasnickolas) [inline resources/views/oneclick-mall-deferred/start.blade.php:44]:
```suggestion
        input type="hidden" name="TBK_TOKEN" value="{{ $resp->token }}" />
```

File: resources/views/oneclick-mall-deferred/start.blade.php [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+        input type="submit" value="Pagar" />
```
Comment (Matiasnickolas) [inline resources/views/oneclick-mall-deferred/start.blade.php:45]:
```suggestion
        input type="submit" value="Inscribir" />
```

File: resources/views/oneclick-mall-deferred/start.blade.php [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+    <h2>Paso 2: Respuesta</h2>
+    <p class="mb-32">
+        Una vez que hayas iniciado la inscripción, aquí encontrarás los datos de respuesta generados por el proceso.
+    </p>
+
+    <x-snippet :content="$resp" />
+
+    <h2>Paso 3: Creación del formulario</h2>
+    <p class="mb-32">
+        Utiliza estos datos de respuesta para generar y presentar un formulario de pago al Tarjetahabiente.
```
Comment (Matiasnickolas) [inline resources/views/oneclick-mall-deferred/start.blade.php:38]:
```suggestion
        Utiliza estos datos de respuesta para generar y presentar un formulario de Inscripción al Tarjetahabiente.
```

File: resources/views/oneclick-mall-deferred/start.blade.php [hunk 9]
```diff
@@ -0,0 +1,77 @@
...
+        transacción. Utilizaremos los siguientes datos para configurar la transacción:
+    </p>
+
+    <div class="mb-32">
+        <x-table :request="$request" />
+    </div>
+
+    <p class="mb-32">
+        Por último, con la respuesta del servicio que confirma la creación de la transacción, procedemos
+        a crear el formulario de pago. Para fines de este ejemplo, haremos visible el campo "token_ws", el cual es
```
Comment (Matiasnickolas) [inline resources/views/oneclick-mall-deferred/start.blade.php:61]:
```suggestion
        a crear el formulario de pago. Para fines de este ejemplo, haremos visible el campo "TBK_TOKEN", el cual es
```

File: resources/views/oneclick-mall-deferred/start.blade.php [hunk 10]
```diff
@@ -0,0 +1,77 @@
...
+        form action="https://webpay3gint.transbank.cl/webpayserver/initTransaction" method="POST">
+        input type="hidden" name="token_ws" value="{{ $resp->token }}" />
+        input type="submit" value="Pagar" />
+        form>
+    </x-snippet>
+
+    <h2 id="example">Ejemplo</h2>
+    <p class="mb-32">
+        Para llevar a cabo una transacción de compra en nuestro sistema, primero debemos crear la
+        transacción. Utilizaremos los siguientes datos para configurar la transacción:
```
Comment (Matiasnickolas) [inline resources/views/oneclick-mall-deferred/start.blade.php:52]:
```suggestion
        Para llevar a cabo una Inscripción en nuestro sistema, primero debemos crearla. Utilizaremos los siguientes datos para configurar la inscripción:
```

**Review by mastudillot — APPROVED** [review]
LGTM

**Review by DiegoHBustamante — APPROVED** [review]

**Pull request merged**